frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/review-and-create-step/review-and-create-step.tsx

```diff
@@ -1,224 +1,128 @@
 import * as React from 'react';
 import { useTranslation } from 'react-i18next';
-import {
-  TextContent,
-  Text,
-  TextVariants,
-  Card,
-  CardBody,
-  AlertVariant,
-  Split,
-  SplitItem,
-} from '@patternfly/react-core';
-import {
-  GreenCheckCircleIcon,
-  YellowExclamationTriangleIcon,
-  BlueInfoCircleIcon,
-  RedExclamationCircleIcon,
-  useFlag,
-} from '@console/shared/src';
+import { TextContent, Text, TextVariants, List, ListItem } from '@patternfly/react-core';
+import { useFlag } from '@console/shared/src';
 import { humanizeBinaryBytes } from '@console/internal/components/utils';
-import { MINIMUM_NODES, NetworkTypeLabels, NO_PROVISIONER } from '../../../../constants';
+import { BackingStorageType } from '../../../../constants/create-storage-system';
+import { NetworkTypeLabels, NO_PROVISIONER } from '../../../../constants';
```

Comment:
Any reason why some constants are in Pascal Case? NetworkTypeLabels

Reply:
NetworkTypeLabels is an enum. I have not created that